compiler-core/src/language_server/tests/snapshots/gleam_core__language_server__tests__action__wrap_anonymous_function_in_anonymous_function.snap

```diff
@@ -0,0 +1,16 @@
+---
+source: compiler-core/src/language_server/tests/action.rs
+expression: "pub fn main() {\n  let f = fn(in) { ception(in) }\n}\n\n"
+---
+----- BEFORE ACTION
+pub fn main() {
+  let f = fn(in) { ception(in) }
```

Comment:
I don't think we should be offering to wrap an anonymous function in another anonymous function

Reply:
fixed

Reply:
Is this snapshot unused now?